Add tests for knative trigger in serverless-controller-manager integration tests (#8312)
kyma-project/kyma: tests/function-controller/README.md

| **APP_TEST_DOMAIN_NAME**                | No       | `test-function`            | The domain name used in the APIRule CR                                                                                                                |
| **APP_TEST_INGRESS_HOST**               | No       | `kyma.local`               | The Ingress host address                                                                                                                              |
| **APP_TEST_DOMAIN_PORT**                | No       | `80`                       | The port of the Service exposed by the API Rule in a given domain                                                                                     |
| **APP_TEST_INSECURE_SKIP_VERIFY**       | No       | `true`                     | The flag that controls whether tests use verification of the server's certificate and the host name to reach the Function                                       |
| **APP_TEST_VERBOSE**                    | No       | `true`                     | The value that controls whether tests log resources that are subject to change | 

> | **APP_TEST_MAX_POLLING_TIME**           | No       | `5m`                       | The maximum period of time that we wait for function to reconfigure after update  | 

Reviewer: ```suggestion
| **APP_TEST_MAX_POLLING_TIME**           | No       | `5m`                       | The maximum period of time in which the Function can reconfigure after an update  | 
```

Reviewer: of sth other than "we" ;) like "the test waits"?

> | **APP_TEST_MAX_POLLING_TIME**           | No       | `5m`                       | The maximum period of time that we wait for function to reconfigure after update  | 

Author: ```suggestion
| **APP_TEST_MAX_POLLING_TIME**           | No       | `5m`                       | The maximum period of time in which the Function must reconfigure after an update  | 
```
